.buildkite/pipeline-utils/agentImages.ts

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
nit, looks like most of the files are snake case

Reply:
[8bd94f9](https://github.com/elastic/kibana/pull/183777/commits/8bd94f919b8c8b6e0e66020bf11d74f3d86bb510)